whiterabbit/src/main/java/org/ohdsi/whiteRabbit/scan/SourceDataScan.java

```diff
@@ -408,6 +413,31 @@ else if (isReal)
 				return "varchar";
 		}
 
+		public Object getFractionUnique() {
+			if (nProcessed == 0 || uniqueCount == 1) {
+				return 0d;
+			}
+			else {
+				double fractionUnique = uniqueCount / (double) nProcessed;
+				if (tooManyValues) {
+					// Due to trimming, the unique count could be an overestimation
+					return String.format("<= %.3f", fractionUnique);
+				} else {
+					return fractionUnique;
+				}
+			}
+
+		}
+
+		public Object getUniqueCount() {

```

Comment:
Try to have a single return type. String seems appropriate. Perhaps rename to `getFormattedUniqueCount()`? You could still create `getUniqueCount()` which returns a `double`.